src/Solana/useSolana.tsx

```diff
@@ -0,0 +1,391 @@
+import { useCallback, useEffect, useState } from 'react'
+import { Buffer } from 'buffer'
+import {
+  Connection,
+  PublicKey,
+  clusterApiUrl,
+  Cluster,
+  VersionedTransaction,
+  AccountInfo,
+} from '@solana/web3.js'
+import { SolanaStatus, useSolanaStatus, useSolanaVars } from './solanaSentinel'
+import * as Currency from '@helium/currency-utils'
+import {
+  Asset,
+  heliumAddressToSolPublicKey,
+  searchAssets,
+  SearchAssetsOpts,
+  sendAndConfirmWithRetry,
+} from '@helium/spl-utils'
+import * as Hotspot from '@helium/hotspot-utils'
+import {
+  AccountLayout,
+  getAssociatedTokenAddress,
+  TOKEN_PROGRAM_ID,
+} from '@solana/spl-token'
+import {
+  entityCreatorKey,
+  init,
+  makerKey,
+} from '@helium/helium-entity-manager-sdk'
+import BigNumber from 'bignumber.js'
+import { getBalance } from '@helium/currency-utils'
+import axios from 'axios'
+import { AnchorProvider, Wallet, Program } from '@coral-xyz/anchor'
+import { HeliumEntityManager } from '@helium/idls/lib/types/helium_entity_manager'
+import { daoKey } from '@helium/helium-sub-daos-sdk'
+
+type Account = AccountInfo<string[]>
+
+// TODO: Get urls for each cluster
+const METAPLEX_URL = 'https://rpc-devnet.aws.metaplex.com/'
```

Comment:
TODO: We need additional metaplex urls?

Reply:
We will for devnet vs mainnet. This will just be the rpc url in mainnet for solana.

Honestly, makers should be able to get and configure their own RPC. Can we just make this configurable?